policies/contractors-invoicing-policy.md

```diff
@@ -4,13 +4,8 @@ The OpenSSL organization is purely remote without any office. The organization h
 
 ## Contractor Invoice Submission
 
-1. Contractors send their invoices via the [Online Form] or send them directly to **finance@openssl.org** in the format and providing the information as detailed below. If an invoice is submitted via the form, notification to the finance@openssl.org mailing list will be sent automatically. 
+1. Contractors send their invoices directly to **finance@openssl.org** in the format and providing the information as detailed below.
```

Comment:
I suspect that "in the format" should be "in pdf format"...

Reply:
"in a universally readable format, such as .pdf," would bring it into line with the procedure document.